.github/workflows/tests-ci.yml

```diff
@@ -62,16 +62,39 @@ jobs:
       - name: Terminate dwn-server
         run: kill $DWN_SERVER_BACKGROUND_PROCESS || true
 
+      - name: Upload test coverage to Codecov
+        uses: codecov/codecov-action@eaaf4bedf32dbdc6b720b63067d99c4d77d6047d # v3.1.4
+        env:
+          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+
       - name: Upload test results to update SDK reports
         uses: actions/upload-artifact@v3
         with:
           name: junit-results
           path: packages/*/results.xml
 
-      - name: Upload test coverage to Codecov
-        uses: codecov/codecov-action@eaaf4bedf32dbdc6b720b63067d99c4d77d6047d # v3.1.4
+      - uses: actions/create-github-app-token@2986852ad836768dfea7781f31828eb3e17990fa # v1.6.2
```

Comment:
@finn-tbd Please add a name that describes this step

Reply:
added